lua/orgmode/parser/section.lua

```diff
@@ -34,8 +34,37 @@ local config = require('orgmode.config')
 ---@field clocked_in boolean
 local Section = {}
 
+---@class SectionTodoKeyword
+---@field node unknown
+---@field type 'TODO'|'DONE'|''
+---@field value string
+local SectionTodoKeyword = {}
+
+---@class NewSectionOptions
+---@field content string[]
+---@field dates Date[]
+---@field level number
+---@field line string
+---@field logbook Logbook
+---@field node table
+---@field own_tags string[]
+---@field parent Section
+---@field priority string
+---@field properties table
+---@field range Range
+---@field root File
+---@field tags string[]
+---@field title string
+---@field todo_keyword_node unknown
+local NewSectionOptions = {}
```

Comment:
There's actually no need to define an empty variable to get type annotations. LSP should know to pick it up even without it.

Reply:
I've removed the two unused variables that I added as part of this contribution